i18n.md

```diff
@@ -0,0 +1,230 @@
+- `sites/kit.svelte.dev/src/routes/+page.svelte`
+- `sites/kit.svelte.dev/src/routes/docs/[slug]/+page.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Deployment.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Features.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Hero.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Intro.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Showcase.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Svelte.svelte`
+- `sites/kit.svelte.dev/src/routes/home/subtitles.vtt`
+- `sites/kit.svelte.dev/src/routes/nav.json/+server.js`
+- `sites/kit.svelte.dev/src/routes/schema.json`
+- `sites/kit.svelte.dev/src/routes/search/+page.svelte`
+
+> Be careful, this list might not be exhaustive.
+
+Additionally, you also might want to change the `lang` attribute in `sites/svelte.dev/src/app.html`.
+
+## Modules and interfaces
+
+The 'Modules' and 'Types' sections of the docs directly refer to the kit package source code. For instance, the documentation [here](https://kit.svelte.dev/docs/modules#$app-environment-building) and [here](https://kit.svelte.dev/docs/types#public-types-adapter) is not written anywhere in the `documentation` folder, but in the kit package source code instead.
+
+If you want the types description to also be translated, then you will need to work on these files:
+- `packages/kit/src/exports/hooks/sequence.js`
+- `packages/kit/src/exports/index.js`
+- `packages/kit/src/exports/node/polyfills.js`
+- `packages/kit/src/exports/public.d.ts`
+- `packages/kit/src/exports/vite/index.js`
+- `packages/kit/src/runtime/app/environment.js`
+- `packages/kit/src/runtime/app/forms.js`
+- `packages/kit/src/runtime/app/navigation.js`
+- `packages/kit/src/runtime/app/stores.js`
+- `packages/kit/src/types/ambient.d.ts`
+- `packages/kit/src/types/private.d.ts`
+- `packages/kit/src/types/synthetic/$env+dynamic+private.md`
+- `packages/kit/src/types/synthetic/$env+dynamic+public.md`
+- `packages/kit/src/types/synthetic/$env+static+private.md`
+- `packages/kit/src/types/synthetic/$env+static+public.md`
+- `packages/kit/src/types/synthetic/$lib.md`
+- `sites/kit.svelte.dev/scripts/types/index.js`
+
+> Be careful, this list might not be exhaustive.
+
+> In these `packages` files, you probably won't be able to use the environment variables for links described [here](#external-links), since it's a different package. It might actually be possible, but we did not investigate too much on the matter.
+
+> You will need to run `npm run generate:types` in the `packages/kit` folder before running `npm run dev` in the `sites/kit.svelte.dev` folder in order to be able to see your changes in dev mode.
+
+## Glossary
+
+In a web development context, some words do not translate well in some languages – or at least in french –, even if they technically have an official translation. This is because people have taken the habit to use the original word in english rather than the translated one. These are called _anglicisms_.
```

Comment:
```suggestion
In a web development context, some words do not translate well in some languages – or at least in French –, even if they technically have an official translation. This is because people have taken the habit to use the original word in English rather than the translated one. These are called _anglicisms_.
```